packages/eslint-config-fbjs/package.json

```diff
@@ -21,9 +21,9 @@
   },
   "peerDependencies": {
     "babel-eslint": "^7.1.1",
-    "eslint": "^3.11.1",
+    "eslint": ">=3.11.1",
```

Comment:
Actually, nevermind. I'm going to just do a wholesale update of this package and ship 2.0. What's in master is already on it's way to 2.0 (.0-alpha.1) so this would all need to be backported and that's no fun.